po/POTFILES.in

```diff
@@ -186,6 +186,7 @@ src/iop/basicadj.c
 src/iop/bilat.c
 src/iop/bilateral.cc
 src/iop/bloom.c
+src/iop/blurs.c
```

Comment:
I'll do it just after merging, but for reference you always have two entries to add, one for the module itself and one for the generated introspection code.